proprietary/tokens/src/common/amsterdam/border.tokens.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Is this generic enough lock down in common design tokens? Maybe we can just hardcode the values in the component level design tokens for now

Reply:
I consider this a shared setting and therefor it would be ok to have this in a common folder